README.md

```diff
@@ -1 +1,59 @@
-# Hello-world
\ No newline at end of file
+# :green_book: Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+# :book: [Hello Microverse project]
```

Comment:
- [ ] Please add Licence section of your README into the table of content. As it is also part of README [template](https://github.com/microverseinc/readme-template) table of content and it is the [requirement of the project](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements) to:

> Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project

Your README:

<img width="274" alt="Screenshot 2023-02-25 at 2 37 15 PM" src="https://user-images.githubusercontent.com/44798044/221350408-26170ca1-a0e4-472f-a6db-173930d46c88.png">


Templete README

<img width="314" alt="Screenshot 2023-02-25 at 2 44 57 PM" src="https://user-images.githubusercontent.com/44798044/221350460-01a46a05-7c37-4410-9877-865cd6ecbe28.png">